store/sqlstore/team_store.go

```diff
@@ -807,7 +754,7 @@ func (s SqlTeamStore) SaveMultipleMembers(members []*model.TeamMember, maxUsersP
 		User  sql.NullString
```

Comment:
This needs to be an empty slice, similar to the changes we have been making so far.